ansible/roles/iroha-docker/tasks/config-gen.yml

```diff
@@ -49,10 +49,11 @@
             - "{{ iroha_genesis_block | from_json | to_json | b64encode }}"
             - "{{ iroha_all_new_nodes | to_json | b64encode }}"
         register: iroha_command_register
-
```

Comment:
you need also update line 31:
```
      - name: Get existing genesis block 2
        set_fact:
          iroha_new_genesis_block: "{{ (iroha_command_register.ansible_facts.docker_container.Output | from_json).result }}"
```
check the whole directory maybe you have a similar line somewhere else.